modules/sbo-categories-of-exposable-binding-data.adoc

```diff
@@ -87,6 +88,8 @@ The following example shows how to expose an entire config map through OLM descr
 This example uses the `path` attribute with a `urn:alm:descriptor:io.kubernetes:ConfigMap` entry to indicate that the path points to the `ConfigMap` service resource.
 
 If you intend to project all the values from a `ConfigMap` service resource, you must specify it as an attribute in the backing service CR. For example, if the attribute is part of the `.spec` section, you can create and use a `specDescriptors` array. Or, if the attribute is part of the `.status` section, you can create and use a `statusDescriptors` array.
+////
+// Add this example once OLM descriptors are supported again.
```

Comment:
`When the OLM descriptors are supported again, add this example.`
or
`Add this example when the OL descriptors are supported again.`